### ISSUE TITLE
Better distribution over videobridges in case of restart

### ISSUE THREAD

**Issue by krombel** [jitsi/jicofo#477]
## Description
In case of jicofo restart it happens that nearly all conferences are pushed to the same videobridge. The update ist to slow to take all videobridges into account.

## Current behavior
After jicofo restart all existing conferences are distributed on one (sometimes two) videobridge(s)

## Expected Behavior
After jicofo restart it should be distributed more equal over all videobridges

## Possible Solution
In case of assigning a conference on a videobridge increase the internal count of assigned conferences (or sth. similar) to not count this videobridge as least loaded anymore.
It would be overwritten by incoming stats reports either way. It would be just to handle this case better. It seems that the stats update is too slow to handle 50 requests in one second which cause conference=>bridge assignments.

## Steps to reproduce
- Have 50 Conferences distributed over 10 videobridges
- restart jicofo
- have all conferences on one (sometimes two) videobridges

# Environment details
latest jicofo with several videobridges

**Comment by bgrozev** [comment]
Jicofo counts newly added participants (in the last 10 seconds) with a certain weight towards the load of a bridge. You can control the weight using this property:
```
org.jitsi.jicofo.BridgeSelector.AVG_PARTICIPANT_PACKET_RATE
```

The default is 500.

**Comment by krombel** [comment]
I don't think this is making any difference in that situation.
More details in my comment on #480

**Comment by bgrozev** [comment]
Fixed in #480/#530
